server/src/main/java/com/sauron/model/entities/util/TransactionDirection.java

```diff
@@ -1,4 +1,4 @@
-package com.sauron.model.entities;
```

Comment:
I don't see a point of moving it. For me, TransactionDirection is not util, it's just model.

Reply:
I agree it isn't util but it isn't enity either. I moved it to just "model" package.

Reply:
:+1: